### ISSUE TITLE
Node.js Technical Steering Committee (TSC) Meeting 2024-07-10

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#1592]
## Time

**UTC Wed 10-Jul-2024 15:00 (03:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 10-Jul-2024 08:00 (08:00 AM) |
| US / Mountain | Wed 10-Jul-2024 09:00 (09:00 AM) |
| US / Central  | Wed 10-Jul-2024 10:00 (10:00 AM) |
| US / Eastern  | Wed 10-Jul-2024 11:00 (11:00 AM) |
| EU / Western  | Wed 10-Jul-2024 16:00 (04:00 PM) |
| EU / Central  | Wed 10-Jul-2024 17:00 (05:00 PM) |
| EU / Eastern  | Wed 10-Jul-2024 18:00 (06:00 PM) |
| Moscow        | Wed 10-Jul-2024 18:00 (06:00 PM) |
| Chennai       | Wed 10-Jul-2024 20:30 (08:30 PM) |
| Hangzhou      | Wed 10-Jul-2024 23:00 (11:00 PM) |
| Tokyo         | Thu 11-Jul-2024 00:00 (12:00 AM) |
| Sydney        | Thu 11-Jul-2024 01:00 (01:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2024-07-10&iso=20240710T1500
* or https://www.wolframalpha.com/input/?i=03PM+UTC%2C+Jul+10%2C+2024+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1CSAVq4Z-U8ORWgx1DxjUHxxpWOMoDnAVIS0T6NaLsKA/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* `npm` is installed wrongly on windows machine [#53538](https://github.com/nodejs/node/issues/53538)

### nodejs/TSC

* New Strategic Initiative on Primordials [#1439](https://github.com/nodejs/TSC/issues/1439)

### nodejs/next-10

* Next 10 - Funding Deep Dive [#273](https://github.com/nodejs/next-10/issues/273)

## Invited

* Antoine du Hamel @aduh95 (voting member)
* Anatoli Papirovski @apapirovski (voting member)
* Benjamin Gruenbaum @benjamingr (voting member)
* Ruben Bridgewater @BridgeAR (voting member)
* Geoffrey Booth @GeoffreyBooth (voting member)
* Gireesh Punathil @gireeshpunathil (voting member)
* James Snell @jasnell (voting member)
* Joyee Cheung @joyeecheung (voting member)
* Chengzhong Wu @legendecas (voting member)
* Marco Ippolito @marco-ippolito (voting member)
* Matteo Collina @mcollina (voting member)
* Michael Dawson @mhdawson (voting member)
* Moshe Atlow @MoLow (voting member)
* Rafael Gonzaga @RafaelGSS (voting member)
* Richard Lau @richardlau (voting member)
* Robert Nagy @ronag (voting member)
* Ruy Adorno @ruyadorno (voting member)
* Paolo Insogna @ShogunPanda (voting member)
* Michaël Zasso @targos (voting member)
* Tobias Nießen @tniessen (voting member)
* Beth Griggs @BethGriggs (regular member)
* Ben Noordhuis @bnoordhuis (regular member)
* Colin Ihrig @cjihrig (regular member)
* Shelley Vohr @codebytere (regular member)
* Rich Trott @Trott (regular member)
* Yagiz Nizipli @anonrig (regular member)
* Joe Sepi @sepi@joesepi.com (Guest - Node.js CPC rep)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: <https://zoom.us/j/611357642>
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **<https://www.youtube.com/c/nodejs+foundation/live>** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Trott** [comment]
If we're truly going to discuss https://github.com/nodejs/node/issues/53538 in the meeting, maybe we should invite some npm folks to attend (and postpone discussion until they can make it, and maybe adjust the meeting time to accommodate them next week or whatever). As @benjamingr mentioned in the issue itself, a discussion within the TSC is unlikely to result in anything useful. But a discussion with the involvement of npm folks *might* at least get the TSC and npm on the same page about one or more of whether this will be fixed, how it will be fixed, when it will be fixed, why it will never be fixed, what everyone should tell users who encounter this so they can fix it.....

Another possibility is for the TSC to "discuss" it but to basically delegate the conversation above (with the npm team) to the Release team. (I'm assuming the Release team is ultimately responsible for the installer. If it's Build instead, then fine, delegate to Build. The point is: Delegate to people who really have a stake. It can even be an ad hoc team to make sure to include @tniessen if he's still invested, and anyone else obviously invested.)

**Comment by richardlau** [comment]
> Another possibility is for the TSC to "discuss" it but to basically delegate the conversation above (with the npm team) to the Release team. (I'm assuming the Release team is ultimately responsible for the installer. If it's Build instead, then fine, delegate to Build. The point is: Delegate to people who really have a stake. It can even be an ad hoc team to make sure to include @tniessen if he's still invested, and anyone else obviously invested.)

The installer doesn't fall into either Release or Build, based on the [responsibilities devolved to those Working Groups](https://github.com/nodejs/TSC/blob/main/WORKING_GROUPS.md).

**Comment by GeoffreyBooth** [comment]
Not sure if this warrants being added to the agenda, but I’d appreciate some guidance on whether people think https://github.com/nodejs/node/pull/53619, unflagging `--experimental-detect-module`, is semver-major. If it is, I guess we should introduce a `--detect-module` flag so that Node <23 can benefit from this feature after it’s unflagged in >=23.

**Comment by mhdawson** [comment]
There is private business to discuss so lets hold the meeting even if it ends up being short.

**Comment by mhdawson** [comment]
PR for minutes - https://github.com/nodejs/TSC/pull/1594
